libs/auth/test/utils.ts

```diff
@@ -7,7 +7,7 @@ export function buildMockCard(): Card {
   return {
     getCardStatus: jest.fn(() => Promise.resolve({ status: 'no_card' })),
     checkPin: jest.fn(),
-    writeUser: jest.fn(),
+    program: jest.fn(),
```

Comment:
Maybe now `clearUserAndData` should be `unprogram` to match?

Reply:
👍 I went back and forth between `unprogram` and `clearUserAndData`, with the latter feeling more descriptive, but being consistent with `program` makes sense